doc/release/RELEASE-NOTES.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
+## Release 1.23.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: November 2, 2024
+
+### Code/API changes
+* introduce throttling for tiles/* endpoins 300/minute 
+
```

Comment:
Pls, add extra space here. Or even better: copy entire block from release template and add your change in `Code/Api changes`. That will avoid possible merge conflicts here